src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -131,4 +133,31 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
       this.openFileDialog();
     }
   }
+
+  getSelectedFile() {
+    console.log('1');
+    console.log(this.uploadedDocument);
+    if (this.uploadedDocument) {
+      console.log('Uploaded document exist.');
```

Comment:
could you please remove these console.log lines?

Reply:
These console logs added to test/debug on Tony's PC using DM services.  Removed them now.   Done.